_content/booklets/awareness-itself_fuang.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Awareness Itself: The Teachings of Ajaan Fuang Jotiko"
+authors:
+  - fuang
+translator: geoff
+external_url: "https://buddho.org/book/awareness-itself/"
```

Comment:
Use the original link instead: https://www.dhammatalks.org/books/#awarenessitself